xskillscore/core/deterministic.py

```diff
@@ -185,9 +184,20 @@ def pearson_r(a, b, dim, weights=None, skipna=False):
     scipy.stats.pearsonr
     xskillscore.core.np_deterministic._pearson_r
 
-    Reference
-    ---------
-    https://en.wikipedia.org/wiki/Pearson_correlation_coefficient
+    Notes
+    -----
+    .. [1] https://en.wikipedia.org/wiki/Pearson_correlation_coefficient
+
+    Examples
+    --------
+    >>> import numpy
+    >>> import xarray
+    >>> from xskillscore import pearson_r
+    >>> a = xr.DataArray(np.random.rand(5, 3, 3),
+                        dims=['time', 'x', 'y'])
+    >>> b = xr.DataArray(np.random.rand(5, 3, 3),
+                        dims=['time', 'x', 'y'])
+    >>> pearson_r(a, b, dim='time')
```

Comment:
Maybe you also indicate here that the result has all dims except time

Reply:
> Maybe you also indicate here that the result has all dims except time
I would suggest your comment goes into a notebook where we can get pretty wordy

Reply:
I updated the `dim` in each docstring to reflect this and added two sentences to the quick start notebook.